centreon/www/include/configuration/configObject/hostgroup/DB-Func.php

```diff
@@ -143,28 +131,297 @@ function disableHostGroupInDB($hg_id = null, $hg_arr = [])
     }
 }
 
+// ---------- UPDATE ----------
+
+function updateHostGroupInDBForCloud(int $hostGroupId, array $submittedValues, bool $increment = false): void
+{
+    global $pearDB, $centreon, $form;
+
+    $request = <<<'SQL'
+            UPDATE hostgroup SET
+                hg_notes = NULL,
+                hg_notes_url = NULL,
+                hg_action_url = NULL,
+                hg_icon_image = NULL,
+                hg_map_icon_image = NULL,
+                hg_rrd_retention = NULL,
+                hg_comment = NULL
+        SQL;
+
+    $bindValues = [];
+
+    if ($submittedValues === []) {
+        $submittedValues = $form->getSubmitValues();
+    }
+
+    if (isset($submittedValues['hg_name'])) {
+        $request .= ', hg_name = :name';
+        $bindValues[':name'] = [
+            PDO::PARAM_STR,
+            $pearDB->escape($submittedValues['hg_name']),
```

Comment:
Used as such, the escape method is useless, since you've created a prepared query.